src/ec/packages/ec-component-blockquote/blockquote.story.js

```diff
@@ -8,18 +9,48 @@ import data from '@ecl/ec-specs-blockquote/demo/data';
 import blockquote from './ecl-blockquote.html.twig';
 import notes from './README.md';
 
+// Labels for the group ids
+// const optionalGroupId = 'Optional elements';
+const requiredGroupId = 'Mandatory elements';
+const useCasesGroup = 'Use cases';
+
+// Preserve the adapted specs.
+const defaultData = { ...data };
+
+// declare your toggled data
+const authorBtnToggler = () => {};
+
+const prepareQuote = prepareData => {
+  // declare your knobs when you format your data
+  const citation = prepareData.citation
+    ? text('Citation', prepareData.citation, requiredGroupId)
+    : false;
+
+  const author = prepareData.author
+    ? text('Author', prepareData.author, requiredGroupId)
+    : false;
+
+  const newQuote = {
+    author,
+    citation,
+  };
+
+  // Return the full object.
+  return newQuote;
+};
+
 storiesOf('Components/Blockquote', module)
   .addDecorator(withKnobs)
   .addDecorator(withCode)
   .addDecorator(withNotes)
   .add(
     'default',
-    () =>
-      blockquote({
-        author: text('Author name', data.author),
-        citation: text('Citation', data.citation),
-      }),
+    () => {
+      button('fake button to enforce groups', authorBtnToggler, useCasesGroup);
+
+      return blockquote(prepareQuote(defaultData));
+    },
     {
-      notes: { markdown: notes, json: data },
+      notes: { markdown: notes, json: defaultData },
```

Comment:
In general i would avoid passing the data with the knobs to the note panel, the logic is that we should have already good data to pass to it as they come form the adapter, then we use the "formatting" function inside the story file only to set the knobs we want and override values for icons and logos, basically. what i fear is that if we pass the knobs this will "call" the knobs again with an unchanged story...This might be already the case in some other component, not sure, but would be better to avoid.